content/en/docs/tasks/administer-cluster/coredns.md

```diff
@@ -16,16 +16,29 @@ This page describes the CoreDNS upgrade process and how to install kube-dns inst
 
 {{% capture steps %}}
 
-## Installing kube-dns with kubeadm
+## About CoreDNS
 
-In Kubernetes 1.11, [CoreDNS](https://coredns.io) has graduated to General Availability (GA)
+[CoreDNS](https://coredns.io) is a flexible, extensible authoritative DNS server that can run on a Kubernetes cluster.
```

Comment:
s/can run on a Kubernetes cluster/can serve as the Kubernetes cluster DNS/

They are different things.

Reply:
I would also continue with a sentence that gives a very brief reason why, like:

CoreDNS includes many functions not available in the previous solution, enabling more customization and control over the cluster DNS behavior.

Reply:
If you add a continuing sentence, make it more specific. Provide an example of the previously unavailable functions, and describe the kinds of customization and control. Pointing to the other DNS topics might help, but you need more meat here, too. Otherwise it sounds like marketing fluff and does not add value.